### PR TITLE
change caption in my file

### PR DESCRIPTION
**Pull request by pgorazda**
updated caption as in #2 

## Done
- create branch
- commit

## Todo
- [ ] merge to main branch
- [ ] delete source branch

cc: @saharM

### CODE REVIEW

File: _slides/33-pgorazda.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I like this caption
```
Comment (pgorazda) [inline _slides/33-pgorazda.md:6]:
```suggestionof
I like this caption a lot 
```

**Review by pgorazda — COMMENTED** [review]
all good

File: _slides/33-pgorazda.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (Jajcarz) [inline _slides/33-pgorazda.md:5]: wrzutka

File: _slides/33-pgorazda.md (hunk 1 above)
Comment (Jajcarz) [reply _slides/33-pgorazda.md:6]: I like to

**Review by Jajcarz — APPROVED** [review]
zatwierdzam

**Pull request merged**